Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -591,14 +591,21 @@ private Task ForceCacheFlushAsync(OperationContext context)
                     try
                     {
                         await Task.Yield();
-                        await _inMemoryCache.FlushAsync(context);
-                        return BoolResult.Success;
+                        return Result.Success(await _inMemoryCache.FlushAsync(context));
                     }
                     finally
                     {
                         Interlocked.Exchange(ref _cacheUpdatesSinceLastFlush, 0);
                         ResetFlushTimer();
                     }
+                }, extraEndMessage: maybeStatistics => {

```

Comment:
The format is weird:)

Reply:
I mean: the opening brace should be on the next line.

---
In reply to: [315389444](https://github.com/microsoft/BuildXL/pull/756#discussion_r315389444) [](ancestors = 315389444)